epregressions/tk_window.py

```diff
@@ -93,7 +93,7 @@ def __init__(self):
 
         # high level GUI configuration
         self.root.geometry('1000x600')
-        self.root.resizable(width=1, height=1)
+        self.root.resizable(width=True, height=True)
```

Comment:
IDE warnings revealed some bad arguments and weird imports, so this file has some miscellaneous cleanups.